docs/multicluster/user-guide.md

```diff
@@ -592,43 +633,150 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
-## Multi-cluster Pod-to-Pod Connectivity
+## NetworkPolicy for Cross-cluster Traffic
 
-Since Antrea v1.9.0, Multi-cluster supports routing Pod traffic across clusters
-through Multi-cluster Gateways. Pod IPs can be reached in all member clusters
-within a ClusterSet. To enable this feature, the cluster's Pod CIDRs must be set in
-ConfigMap `antrea-mc-controller-config` of each member cluster like the example
-below. Note, **Pod CIDRs must not overlap among clusters to enable cross-cluster
-Pod-to-Pod connectivity**.
+Antrea-native policies can be enforced on cross-cluster traffic in a ClusterSet.
+To use this feature set, make sure to check the Antrea controller and agent ConfigMaps
+and confirm that `enableStretchedNetworkPolicy` is set to `true` in addition to enabling
```

Comment:
Just to confirm the feature gate is required by the egress ToService rules too?

Reply:
Yes